src/Categories/Functor/Coalgebra.agda

```diff
@@ -19,3 +19,12 @@ open F-Coalgebra
 -- Given a F-Coalgebra F, one can apply F to it to obtain an new 'iterated' F-Coalgebra
 iterate : ∀ {o ℓ e} {C : Category o ℓ e} {F : Endofunctor C} → F-Coalgebra F → F-Coalgebra F
 iterate {F = F} F-alg = record { A = Functor.F₀ F $ A F-alg ; α = Functor.F₁ F $ α F-alg }
+
+record F-Coalgebra-Morphism {o ℓ e} {C : Category o ℓ e} {F : Endofunctor C} (X Y : F-Coalgebra F) : Set (ℓ ⊔ e) where
+  open Category C
+  module X = F-Coalgebra X
```

Comment:
Can you make these 2 modules 1) private, and 2) qualified with `using` clauses so that they are smaller?